src/google/protobuf/compiler/csharp/csharp_enum.cc

```diff
@@ -49,8 +49,8 @@ namespace protobuf {
 namespace compiler {
 namespace csharp {
 
-EnumGenerator::EnumGenerator(const EnumDescriptor* descriptor) :
-    SourceGeneratorBase(descriptor->file()),
+EnumGenerator::EnumGenerator(const EnumDescriptor* descriptor, const Options* file_options) :
```

Comment:
file_options is slightly misleading here - that sounds like it's a DescriptorProtos.FileOptions. I'd call it generator_options or just options.